Automattic/delphin
# Search: Fix previous search terms not cleared

Originally reported in #1066. During usability testing:
>When searching for a new domain the previous search terms were saved. Some users didn't notice this and unknowingly added their new search terms without deleting the previous terms.

#### Before
![old-flow](https://cloud.githubusercontent.com/assets/699132/21395640/843751f0-c79d-11e6-8b54-59de09f83744.gif)

#### After
![new-flow](https://cloud.githubusercontent.com/assets/699132/21395645/8ae4f016-c79d-11e6-9630-9ff8d5826918.gif)


### Testing
1. Open https://delphin.live/?branch=fix/1066-previous-search-term-not-cleared.
2. Log in.
3. Go to homepage and type any search term.
4. Go to My Domains.
5. Click on Find a new domain.
6. Old keywords and search results should be cleared.

### Review

@Automattic/sdev-feed
